packages/terra-arrange/src/Arrange.jsx

```diff
@@ -55,13 +69,13 @@ const Arrange = ({
 
   return (
     <div {...customProps} className={cx('arrange', customProps.className)} >
-      <div className={cx('fit', align || alignFitStart)}>
+      <div {...fitStartAttributes} className={cx('fit', align || alignFitStart)}>
```

Comment:
You should probably add `fitStartAttributes.className` to  the className here. Same for the others.